### ISSUE TITLE
Android - Concierge - Blank error message badge appears when tapping "Call me" button in request call  modal

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#9281]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in to NewDot with any account
2. Find the Concierge chat
3. Tap the phone icon
4. Tap Call

## Expected Result:
The badge displayed with error message inside after tap Call

## Actual Result:
The badge displayed without error message inside after tap Call

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Android

**Version Number:** 1.1.70-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** applausetester+0901abb@applause.expensifail.com / Feya87Katya
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
![Bug5593526_Screenshot_20220601-160714_New_Expensify](https://user-images.githubusercontent.com/43996225/171501377-95f4dad2-b941-4d0f-950d-e985068baf6f.jpg)

**Expensify/Expensify Issue URL:** 
**Issue reported by:**  Applause internal team
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by srikarparsi** [comment]
Hey @kavimuru I'm unable to reproduce this issue locally, would you mind trying again?

**Comment by kavimuru** [comment]
@srikarparsi I am still able to reproduce in the version 1.1.70-3

https://user-images.githubusercontent.com/43996225/171741333-b7b52fdd-fae7-4dcc-9370-ee597aabb18f.mp4

**Comment by NehaSantani** [comment]
Hey,
The error appears in web as well !
In Inbox.js file, below line is used to display message 
```
  Growl.error(result.message, 3000);
```
However, when we are tapping "Call me" we are getting error from server and the result is not having the expected keys!
 
- we are getting  error-response as `Error {jsonCode: 'unableToRetry'}`.
- There is no "message" key coming from server in response.

**Comment by srikarparsi** [comment]
Hey, thank you for the details. Do you think you can send a screen recording of the web app not working? My android emulator is currently not working and I wasn't able to reproduce it on web either.

**Comment by NehaSantani** [comment]
Sure!

https://user-images.githubusercontent.com/4989101/172525088-01b78bea-e393-4d87-b4c3-ed89afca5b37.mov

**Comment by srikarparsi** [comment]
Hey, thank you for the screen recording. I did some testing and was able to reproduce the error when using the production web proxy but when I used a local dev environment, staging or production, I didn't face the error. Because of this, I believe it's an error with the proxy on web at least and not the code.

https://user-images.githubusercontent.com/48188732/172953862-377e5ec1-bb25-410d-8254-7c8162244107.mov

I am also unable to reproduce this issue on android so I will close out the issue for now.
